packages/material-ui/src/Avatar/Avatar.js

```diff
@@ -107,6 +107,9 @@ function useLoaded({ src, srcSet }) {
       }
       setLoaded('error');
     };
+    if (crossOrigin != null) {
```

Comment:
Changed to nullish comparison because the empty string (which would evaluate to false in `if (crossOrigin)`) is different to not specifying `crossOrigin`:

> If crossOrigin is an empty string (""), the anonymous mode is selected.

-- https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/crossOrigin#value

Reply:
How about direct assignment without judgment?

Reply:
> But how about direct assignment without judgment?

Right. Seems safe here so let's just assign it unconditionally.